control_tlv320aic3206.h

```diff
@@ -0,0 +1,128 @@
+	Purpose: Control module for Texas Instruments TLV320AIC3206 compatible with Teensy Audio Library
+
+	License: MIT License.  Use at your own risk.
+
+	Note that the TLV320AIC3206 has a reset that should be connected to microcontroller.
+		This code defaults to Teensy Pin 21, which was used on Tympan Rev C.
+		You can set the reset pin in the constructor
+		
+	Note that this can configure the AIC for 32-bit per sample I2S transfers or 16-bit per sample.
+		Defaults to 16 bits per sample to allow it to work with default Teensy audio library.
+	
+	Example Usage for Teensy:
+	
+		#include <Audio.h>	//Teensy Audio Library
+		int AIC3206_RESET_PIN=21;     //What pin is your AIC3206 reset pin connected to?
+		AudioControlTLV320AIC3206     audioHardware(AIC3206_RESET_PIN);
+		AudioInputI2S                 i2s_in;        //Digital audio *from* the AIC.
+		AudioOutputI2S                i2s_out;       //Digital audio *to* the AIC.
+		
+		AudioConnection               patchCord1(i2s_in, 0, i2s_out, 0); //connect left input to left output
+		AudioConnection               patchCord2(i2s_in, 1, i2s_out, 1); //connect right input to right output
+		
+		void setup(void)
+		{
+		  //begin the serial comms (for debugging)
+		  Serial.begin(115200);  delay(500);
+		  Serial.println("AudioPassThru: Starting setup()...");
+
+		  //allocate the dynamic memory for audio processing blocks
+		  AudioMemory(10); 
+
+		  //Enable the TLV32AIC3206 to start the audio flowing!
+		  audioHardware.enable(); // activate AIC
+
+		  //Choose the desired input
+		  audioHardware.inputSelect(AIC3206_INPUT_IN1);       // use Input 1
+
+		  //Set the desired volume levels
+		  audioHardware.volume_dB(0);                   // headphone amplifier.  -63.6 to +24 dB in 0.5dB steps.
+		  audioHardware.setInputGain_dB(10.0); // set input volume, 0-47.5dB in 0.5dB setps
+		}
+
+		void loop(void)
+		{
+		  // Nothing to do
+		}
+ */
+
+#ifndef control_tlv320aic3206_h_
+#define control_tlv320aic3206_h_
+
+#include "AudioControl.h"
+#include <Arduino.h>
+
+//convenience names to use with inputSelect() to set whnch analog inputs to use
+#define AIC3206_INPUT_IN1            1   //uses IN1
+#define AIC3206_INPUT_IN2      		 2   //uses IN2 analog inputs
+#define AIC3206_INPUT_IN3    		 3   //uses IN3 analog inputs
+#define AIC3206_INPUT_IN3_MICBIAS	 4   //uses IN3 analog inputs *and* enables mic bias
+
+//convenience names to use with outputSelect()
+#define AIC3206_OUTPUT_HEADPHONE_JACK_OUT 1
+#define AIC3206_OUTPUT_LINE_OUT 2
+#define AIC3206_OUTPUT_HEADPHONE_AND_LINE_OUT 3
+
+//names to use with setMicBias() to set the amount of bias voltage to use
+#define AIC3206_MIC_BIAS_OFF             0
+#define AIC3206_MIC_BIAS_1_25            1
+#define AIC3206_MIC_BIAS_1_7             2
+#define AIC3206_MIC_BIAS_2_5             3
+#define AIC3206_MIC_BIAS_VSUPPLY         4
+#define AIC3206_DEFAULT_MIC_BIAS AIC3206_MIC_BIAS_2_5
+
+#define BOTH_CHAN 0
+#define LEFT_CHAN 1
+#define RIGHT_CHAN 2
```

Comment:
Can these 3 macros be moved into the .cpp file (best option), or put inside the class, or at least have "AIC3206_" prepended?